### ISSUE TITLE
 ERROR: Failure in AnalysisManager.run and ValueError: No JSON object could be decoded

### ISSUE THREAD

**Issue by foolishhare** [cuckoosandbox/cuckoo#2548]
Thanks for creating an issue! But first: did you read our community guidelines?
https://cuckoo.sh/docs/introduction/community.html

**Comment by doomedraven** [comment]
best issue ever :D

**Comment by RicoVZ** [comment]
Hi foolishhare,

Thanks for posting an issue. It would have been useful if some more info was included.

Could you share the cuckoo.log for this analysis?

**Comment by foolishhare** [comment]
> best issue ever :D

Sorry for confusing)

**Comment by doomedraven** [comment]
np :) that was funny to see empty issue :P

**Comment by foolishhare** [comment]
> Hi foolishhare,
> 
> Thanks for posting an issue. It would have been useful if some more info was included.
> 
> Could you share the cuckoo.log for this analysis?

So, what the magical problem happened?

**Comment by foolishhare** [comment]
Trying to debug get a little bit another output.
![image](https://user-images.githubusercontent.com/14123319/48257476-48b6f580-e41b-11e8-9d5e-6e83f650cb38.png)

**Comment by doomedraven** [comment]
is your os using english lang or different? could be that if you use UA lang or any other

**Comment by foolishhare** [comment]
That`s my dump file
[dump.zip](https://github.com/cuckoosandbox/cuckoo/files/2565512/dump.zip)

**Comment by doomedraven** [comment]
maybe not related but prefer to make sure, you are using the esx, but using 192.168.56.x which is the vbox default range, did you set that range for esx?

**Comment by foolishhare** [comment]
> is your os using english lang or different? could be that if you use UA lang or any other

My os using englis
![image](https://user-images.githubusercontent.com/14123319/48258067-eb23a880-e41c-11e8-9d06-24a3964a69bd.png)

**Comment by doomedraven** [comment]
no no, the windows vm lang

**Comment by foolishhare** [comment]
> maybe not related but prefer to make sure, you are using the esx, but using 192.168.56.x which is the vbox default range, did you set that range for esx?

![image](https://user-images.githubusercontent.com/14123319/48258207-39d14280-e41d-11e8-9040-1c7eb6acae9d.png)

**Comment by doomedraven** [comment]
yes yes i saw that posted in one of the comments, but the default network range is vbox, im not sure which one esx uses

**Comment by doomedraven** [comment]
can you post the output of `ifconfig`?

**Comment by foolishhare** [comment]
> no no, the windows vm lang

It uses RU. Does its matter?

**Comment by foolishhare** [comment]
> can you post the output of `ifconfig`?

![image](https://user-images.githubusercontent.com/14123319/48258480-ced43b80-e41d-11e8-8d0b-acf95c2f7fbd.png)

**Comment by doomedraven** [comment]
looks like, it not able to handle ru lang correctly
can you try with windows vm with eng lang? just to make sure what is the problem

**Comment by foolishhare** [comment]
> looks like, it not able to handle ru lang correctly
> can you try with windows vm with eng lang? just to make sure what is the problem

Ok, start trying

**Comment by foolishhare** [comment]
> looks like, it not able to handle ru lang correctly
> can you try with windows vm with eng lang? just to make sure what is the problem

Will it be sufficient if I just change the language settings in the OS?

**Comment by doomedraven** [comment]
yes if your version allows change lang package, make sure username doesn't contain ru chars also or any other path
